### PR TITLE
feat: sketch file format

### LINKED ISSUE #76: Basic support for Sketch File Format 121
- basic support for parsing/serialized https://github.com/sketch-hq/sketch-file-format (just `document.json` for now with all content)
- `*.sketch` parser for `json-schema-ref-parser`

### PR DESCRIPTION
**Pull request by raveclassic**
closes #76

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -78,4 +83,9 @@ const specLikeCodec = union([
 	type({
 		asyncapi: literal('2.0.0'),
 	}),
+	type({
+		meta: type({
```
Comment (sutarmin) [inline src/index.ts]: I don't understand the naming. Why this is called `meta`? Shouldn't it be 'sketch'?

Comment (raveclassic) [reply src/index.ts]: It's part of sketch file format spec

Comment (sutarmin) [reply src/index.ts]: I mean, how do we know later, that this one is related to sketch? Maybe leave a comment or smth?

**Comment by sutarmin** [comment]
@raveclassic conflict

File: src/index.ts (hunk 1 above)
Comment (raveclassic) [reply src/index.ts]: Ok, I'll a comment. Please proceed with the rest of the review, don't stop on a single comment.

Comment (raveclassic) [reply src/index.ts]: adde a comment

**Comment by raveclassic** [comment]
@sutarmin Yeah, I haven't noticed your approval, thanks

**Pull request merged**